This code describes how we compared data from the current study to data from previous studies.


DE genes according to the MAST test and ordered by fold-change (FC), shown as standardized (Z) log2(RPKM)
<img src="https://github.com/KunFang93/SplMeso_PDAC_NC/avg_expr/blob/main/EDF7e.png" width="400">


This is the spl_meso genes, ordered by the log2ratio in the current study (PDAC/normal), and shown as normalized gene expression level; Z(log2(RPKM)). On the right hand size, is FibroXplorer and TabulaSapiens data. TabulaSapiens dataset was converted from hd5 into R objects using SeuratDisk package. Mouse/human gene mappings were always by homologene resource.
<img src="https://github.com/KunFang93/SplMeso_PDAC_NC/avg_expr/blob/main/EDF7j.png" width="400">


Plots that include more details as to the data sources are produced by this script, and will resemble:
<img src="https://github.com/KunFang93/SplMeso_PDAC_NC/avg_expr/blob/main/EDF7e_full.png" width="400">
<img src="https://github.com/KunFang93/SplMeso_PDAC_NC/avg_expr/blob/main/EDF7j_full.png" width="400">

